src/pages/cookie-settings.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Cookie Settings"
+date: "2022-01-25"
+---
+
+# Cookie settings
+
+### About this cookie policy
+
+This Cookie Policy explains what cookies are and how we use them. You should read this policy to understand what cookies are, how we use them, the types of cookies we use i.e, the information we collect using cookies and how that information is used and how to control the cookie preferences. For further information on how we use, store and keep your personal data secure, see our Privacy Policy.
+
+You can at any time change or withdraw your consent from the Cookie Declaration on our website. Learn more about who we are, how you can contact us and how we process personal data in our Privacy Policy. Your consent applies to the following domains: docs.snowplowanalytics.com
+
+### What are cookies?
+
+Cookies are small text files that are used to store small pieces of information. The cookies are stored on your device when the website is loaded on your browser. These cookies help us make the website function properly, make the website more secure, provide better user experience, and understand how the website performs and to analyze what works and where it needs improvement.
+
+### How do we use cookies?
+
+As most of the online services, our website uses cookies first-party and third-party cookies for a number of purposes. The first-party cookies are mostly necessary for the website to function the right way, and they do not collect any of your personally identifiable data.
+
+The third-party cookies used on our websites are used mainly for understanding how the website performs, how you interact with our website, keeping our services secure, providing advertisements that are relevant to you, and all in all providing you with a better and improved user experience and help speed up your future interactions with our website.
+
+The below list details the cookies used in our website.
+<table>
+  <thead>
+    <tr>
+      <th scope="col">Cookie</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="2">Necessary</th>
+    </tr>
+    <tr>
+      <td>cookie-preferences</td>
+      <td>Ironically, to respect your cookie preferences we have to set a cookie for storing and processing them.</td>
```

Comment:
Makes me wonder why these plugins don’t use localStorage...

Reply:
Maybe they want to be by default sharing the preferences with the server.